src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -31,7 +29,9 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
 }) => {
   const firebase = useFirebase();
 
-  const history = useHistory();
+  const venue = useSelector(
+    (state) => state.firestore.ordered.currentVenue?.[0]
+  );
```

Comment:
I believe we already have a `const` selector for this pattern. Best to use that.